app/src/main/java/com/github/se/travelpouch/ui/dashboard/EditActivity.kt

```diff
@@ -119,9 +119,25 @@ fun EditActivity(navigationActions: NavigationActions, activityViewModel: Activi
                   },
                   enabled = true,
                   label = { Text("Date") },
+                  placeholder = { Text("DD/MM/YYYY") },
                   visualTransformation = DateVisualTransformation(),
                   keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
-                  modifier = Modifier.testTag("dateField"))
+                  modifier = Modifier.fillMaxWidth().testTag("dateField"),
+                  trailingIcon = {
+                    IconButton(
+                        onClick = {
+                          dateTimeUtils.showDatePicker(context) { selectedDate ->
+                            date =
+                                selectedDate.replace(
```

Comment:
As was explained to me in the DMs, this is weird and seems redundent but there's a good reason for this inelegant and counter intuitive code. Visual Transformer messes with the format, it's out of scope to refactor it at this stage so this band aid will suffice for now.

Reply:
Yes like we say in the DM, it is not really elegant to do this like that. I will check if I can do it better.

Reply:
I haven't found a way to make it more elegant without adding more complexity. We will keep it like this.